### PR TITLE
Added jupyternotebook and dataset for ML experimentation demo

### PR DESCRIPTION
**Pull request by vinodhini-sd**
Added ML experimentation demo for structured data

### CODE REVIEW

File: 10-ml-experimentation-with-lakeFS-structured-data/README.md
Comment (rmoff) [inline 10-ml-experimentation-with-lakeFS-structured-data/README.md:1]: is it worth expanding the README to say how to run the notebook, and/or to link to our [docs pages](https://docs.lakefs.io/use_cases/reproducibility.html)  ?

**Comment by rmoff** [comment]
can you also add an entry to the root README in the repo too please

**Review by rmoff — CHANGES_REQUESTED** [review]

File: 10-ml-experimentation-with-lakeFS-structured-data/README.md
Comment (vinodhini-sd) [reply 10-ml-experimentation-with-lakeFS-structured-data/README.md:1]: Yep, updated the readme with detailed instructions.

**Comment by vinodhini-sd** [comment]
@rmoff - TODO: Add link to the blog once it is up on the website.

**Review by rmoff — APPROVED** [review]

**Pull request merged**